lib/js/jscore.nim

```diff
@@ -110,3 +110,15 @@ proc parse*(l: JsonLib, s: cstring): JsRoot {.importcpp.}
 since (1, 5):
   func debugger*() {.importjs: "debugger@".}
```

Comment:
where is `"debugger@"` documented and what does it mean? 

manual mentions `"createDevice(@)"` but not `"createDevice@"`

Reply:
```console
$ cat example.nim
func debugger*() {.importjs: "debugger".}
debugger()

$ nim js example.nim
Error: `importjs` for routines requires a pattern

$
```
:shrug:

Reply:
* but shouldn't it be `"debugger(@)" ? `"debugger@"` seems wrong and may (in future) turn into a CT error
* in future work we should support `importjs: "debugger"` though

Reply:
`"debugger(@)"` emits `debugger()` but must be `debugger`.

Reply:
can you document this in a followup PR?

Reply:
can't you use `importc: "debugger"`?

Reply:
It looks like a small bug on importjs tho. :shrug:

Reply:
i added a note in https://github.com/nim-lang/RFCs/issues/315 so we don't lose track